Anlab.Mvc/Controllers/AdminController.cs

```diff
@@ -35,23 +35,24 @@ public AdminController(ApplicationDbContext dbContext, UserManager<User> userMan
         [Authorize(Roles = RoleCodes.Admin)]
         public async Task<IActionResult> Index()
         {
-            var adminRole = await _roleManager.FindByNameAsync(RoleCodes.Admin);
-            var userRole = await _roleManager.FindByNameAsync(RoleCodes.User);
+            // TODO: find better way than super select
+            var usersInRoles = _dbContext.Users.Select(u => new UserRolesModel { User = u });
```

Comment:
I'm certain you could extend the IdentityRole model to include a Users property.

Reply:
Or maybe they have a RoleManager?

Reply:
@jpknoll Yea I tried w/ the role manager and that worked as well, but this was slightly more efficient since otherwise we'd have to enumerate each role (instead of enumerating each user).  But maybe that's a better way to go since most of what we care about is just admin role users